src/Analysis/Ast/Impl/Analyzer/Handlers/IImportedVariableHandler.cs

```diff
@@ -0,0 +1,26 @@
+// Copyright(c) Microsoft Corporation
+// All rights reserved.
+//
+// Licensed under the Apache License, Version 2.0 (the License); you may not use
+// this file except in compliance with the License. You may obtain a copy of the
+// License at http://www.apache.org/licenses/LICENSE-2.0
+//
+// THIS CODE IS PROVIDED ON AN  *AS IS* BASIS, WITHOUT WARRANTIES OR CONDITIONS
+// OF ANY KIND, EITHER EXPRESS OR IMPLIED, INCLUDING WITHOUT LIMITATION ANY
+// IMPLIED WARRANTIES OR CONDITIONS OF TITLE, FITNESS FOR A PARTICULAR PURPOSE,
+// MERCHANTABILITY OR NON-INFRINGEMENT.
+//
+// See the Apache Version 2.0 License for specific language governing
+// permissions and limitations under the License.
+
+using System.Collections.Generic;
+using Microsoft.Python.Analysis.Modules;
+using Microsoft.Python.Analysis.Values;
+
+namespace Microsoft.Python.Analysis.Analyzer.Handlers {
+    internal interface IImportedVariableHandler {
+        IEnumerable<string> GetMemberNames(PythonVariableModule variableModule);
```

Comment:
After rereading this PR and checking references/impls, this function actually only ever returns what gets copied in a `*` import, so `GetMemberNames` is actually a misnomer and should probably be renamed to better reflect that, like `GetStartImportMemberNames` or similar.